jplag.frontend.text/src/main/java/de/jplag/text/ParserAdapter.java

```diff
@@ -1,23 +1,36 @@
 package de.jplag.text;
 
-import java.io.File;
-import java.io.FileInputStream;
-import java.util.HashMap;
-import java.util.Map;
-
-import antlr.Token;
-
 import de.jplag.AbstractParser;
 import de.jplag.TokenConstants;
 import de.jplag.TokenList;
+import edu.stanford.nlp.ling.CoreLabel;
+import edu.stanford.nlp.pipeline.CoreDocument;
+import edu.stanford.nlp.pipeline.StanfordCoreNLP;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
 
 public class ParserAdapter extends AbstractParser {
 
     private final Map<String, Integer> tokenTypes = new HashMap<>();
+    private final StanfordCoreNLP pipeline;
     private int tokenTypeIndex = 1; // 0 is FILE_END token, SEPARATOR is not used as there are no methods.
 
     private TokenList tokens;
     private String currentFile;
+    private int currentLine;
+    private int currentLineIndex;
```

Comment:
This is rather the rather the `globalCharIndex`, right? At least from the name it is not clear to me what the purpose of the attribute is.

Reply:
It's the global index of the current line break. I'm not sure about a better name, but I'll add a comment explaining it. If you have a better name, please let me know.

Reply:
Then I would name it `currentLineBreakIndex` to make things clear.